pkgs/applications/science/logic/klee/default.nix

```diff
@@ -128,9 +126,9 @@ in stdenv.mkDerivation rec {
       that matches a computed test input, including setting up files, pipes,
       environment variables, and passing command line arguments.
     '';
-    homepage = "https://klee.github.io/";
+    homepage = "https://github.com/klee/klee";
```

Comment:
Why the change from the website to the GitHub for the homepage?

Reply:
Consistency with klee-uclibc

Reply:
Oh, though would most people need to know the GitHub or would they want to visit the website for finding info on klee?